### PR TITLE
Welcome bot added

### PR DESCRIPTION
**Pull request by Ankita297**
## Related Issue 
I have added a github action . If contributor makes his/her first PR or open his/her first issue . This action will greet them .

Closes: #161 

### Describe the changes you've made

Give a clear description what modifications you have made

## Type of change

What sort of change have you made:

- [x ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [x ] Enhancement in the existing features
- [x ] Other

## Mention any unusual behaviour of your code (Write NA if not)
Any unusual behaviour of your code

## Checklist:

- [ x] My code follows the guidelines of this project.
- [ x] I have performed a self-review of my own code.
- [ x] I have commented my code, particularly whereever it was hard to understand.
- [ x] My changes generate no new warnings.
- [ x] I have verified/tested my code by running it locally.

## Additional Info (optional)
Any additional information you want to give
New issue · Issue #1 · Ankita297_Story - Google Chrome 22-04-2021 00_53_11

### CODE REVIEW

**Comment by Ankita297** [comment]
@sagar-barapatre @Namanl2001 Please review my PR .

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Hi 😄, thanks for creating your first issue at MERN-GURUJI-DEV, do read and follow the  [Contribution Guidelines](https://github.com/Namanl2001/MERN-Gurujii-dev/blob/master/contributing.md) while contributing.Someone from our team will get back here soon '
```
Comment (Namanl2001) [inline .github/workflows/greetings.yml:12]:
```suggestion
        issue-message: 'Hi 😄, thanks for creating your first issue at MERN-GURUJI-DEV, do read and follow the  [Contribution Guidelines](https://github.com/Namanl2001/MERN-Gurujii-dev/blob/master/contributing.md) while contributing. Someone from our team will get back here soon.'
```

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        pr-message: 'Thank you for your pull request and welcome to our community! We will soon be getting back to you. Your patience will be greatly appreciated!Thanks! 🥳'
```
Comment (Namanl2001) [inline .github/workflows/greetings.yml:13]:
```suggestion
        pr-message: 'Thank you for your pull request, and welcome to our community! We will be getting back to you soon. Your patience will be greatly appreciated! Thanks! 🥳'
```

**Review by Namanl2001 — COMMENTED** [review]
Small nits else look good to me.

**Comment by Ankita297** [comment]
@Namanl2001 changes done

**Pull request merged**